apprise/plugins/telegram.py

```diff
@@ -307,19 +307,6 @@ class NotifyTelegram(NotifyBase):
             " -",
             {},
         ),
-        # convert pre tags to code (supported by Telegram)
-        (
-            re.compile(r"<\s*pre([^a-z0-9>][^>]*)?>", (re.I | re.M | re.S)),
-            "{}<code>",
-            {"html": "\r\n"},
-        ),
-        (
-            re.compile(
-                r"<\s*/\s*pre([^a-z0-9>][^>]*)?>", (re.I | re.M | re.S)
-            ),
-            "</code>{}",
-            {"html": "\r\n"},
-        ),
```

Comment:
It appears this and a few lines in thte test_plugin_telegram.py are the only changes in this PR.   Is there a reason for all of the other file mainipulation and re-formatting?  It just makes the change itself a bit confusing.

Reply:
Apologies, I think my editor accidentally ran the wrong formatter! Reverted the formatting now :)